SPECS/systemd/systemd.spec

```diff
@@ -260,6 +260,7 @@ Provides:       /bin/systemctl
 Provides:       /sbin/shutdown
 Provides:       syslog
 Provides:       systemd-units = %{version}-%{release}
+%{?azure:Obsoletes:      systemd-bootstrap <= %{version}-%{release}}
```

Comment:
where is the `azure` macro defined?

Reply:
https://github.com/microsoft/CBL-Mariner/blob/0ffbc7a70cf9e3126cdf43a9798b17f71f44c345/SPECS/systemd/systemd.spec#L2
I think the idea is we probably want to move this to a macro file in the future? But for now its hard-coded